@flexa/fallback state (#163)
NethermindEth/Forta-Agents: Flexa-Agents/Fallback-State/README.md

# Fallback State Agent

## Description

> This agent detects the contract entering a fallback state

Reviewer: ```suggestion
This agent detects when `FlexaCollateralManager` enters a fallback state
```
Suggestion for a slight re-wording as well as specifying which contract is being referred to.

## Supported Chains

- Ethereum

## Alerts

- FLEXA-4
  - Fired whenever the contract enters a fallback state
  - Severity is always set to "Info"
  - Type is always set to "Info"
  - Metadata fields
    - blockTimestamp: Timestamp of the block

>     - blockNumber: Block number of the block

Author: I'll remove metadata since it's not needed for this agent as Lazaro mentioned below. https://github.com/NethermindEth/Forta-Agents/pull/163#discussion_r821126700
